Explain how to disable editable placeholders (#88)
redpanda-data/docs-site: meta-docs/STYLE-GUIDE.adoc

== Placeholders

Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. Here are some rules for using placeholders:

- Wrap placeholders in angled brackets like so `<placeholder>`.
- Use a descriptive name as the placeholder value. For example `<broker-id>`.
- Use lowercase characters.
- Separate words with a dash.

> - Do not use possessives or instructions as values, such as `<replace-with or my-value>`.

Reviewer: ```suggestion
- Do not use possessives or instructions as values, such as `<replace-with> or <my-value>`.
```

== Placeholders

> Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. A placeholder has a descriptive name as its value. Always separate words with a dash. *Do not* use possessives or instructions as values, such as replace-with or my-value.

Author: ```suggestion
Placeholders in sample code and commands represent values that the user must replace. Placeholders in example output can also represent other values that vary. Here are some rules for using placeholders:

- Use a descriptive name as the placeholder value. For example `<broker-id>`.
- Use lowercase characters.
- Separate words with a dash.
- Do not use possessives or instructions as values, such as `<replace-with or my-value>`.
```

rpk topic create <topic-name>
----

Replace `<topic-name>` with the name that you want to give your new topic.

=== Editable placeholders

> The Redpanda documentation UI includes a custom script that makes placeholder values in code blocks editable.

Author: ```suggestion
The Redpanda documentation UI includes a custom script that makes placeholder values in code blocks editable. Inline code fences are not editable.
```
